docs/operator-manual/argocd-cm.yaml

```diff
@@ -331,3 +331,43 @@ data:
     - url: https://mycompany.splunk.com?search={{.metadata.namespace}}
       title: Splunk
       if: kind == "Pod" || kind == "Deployment"
+
+  extension.config: |
```

Comment:
If I change this config, will the API server automatically pick up the change? Or will I have to restart the pod?

Reply:
not 100% sure but I believe you have to restart. Like in every other config in that cm.

Reply:
added a comment in the docs